pulsar-client/src/main/java/org/apache/pulsar/client/impl/PulsarChannelInitializer.java

```diff
@@ -72,7 +72,10 @@ public PulsarChannelInitializer(ClientConfigurationData conf, Supplier<ClientCnx
         if (tlsEnabled) {
             if (tlsEnabledWithKeyStore) {
                 AuthenticationDataProvider authData1 = conf.getAuthentication().getAuthData();
-
+                if (conf.getTlsTrustStorePath() == null) {
```

Comment:
We should check if it is an empty String? Empty string also does not make any sense here.

Reply:
fixed.